File: acd/java/ficheros/src/main/java/com/alexcoding/ficheros/Entities/Jugador.java
package com.alexcoding.ficheros.Entities;

public class Jugador {
    protected String name;
    protected int edad;

    public Jugador(String name, int edad){
        this.name = name;
        this.edad = edad;
    }


    public String getName() {
        return this.name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public int getEdad() {
        return this.edad;
    }

    public void setEdad(int edad) {
        this.edad = edad;
    }
    
}

File: acd/rust/ficheros/src/entities/jugador.rs
// definir estructura del jugador
pub struct Jugador {
    name: String,
    edad: u32,
}

// implementa metodos para el jugador
impl jugador {
    // constructor
    pub fn new(name: String, edad: u32) -> self {
        jugador { name, edad }
    }

    // Método para obtener el nombre
    pub fn get_name(&self) -> &str {
        &self.name
    }

    // Método para establecer el nombre
    pub fn set_name(&mut self, name: String) {
        self.name = name;
    }

    // Método para obtener la edad
    pub fn get_edad(&self) -> u32 {
        self.edad
    }

    // Método para establecer la edad
    pub fn set_edad(&mut self, edad: u32) {
        self.edad = edad;
    }
}
